src/Components/Web/src/Forms/ExpressionFormatting/ReverseStringBuilder.cs

```diff
@@ -0,0 +1,163 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Buffers;
+using System.Globalization;
+
+namespace Microsoft.AspNetCore.Components.Forms;
+
+internal unsafe ref struct ReverseStringBuilder
```

Comment:
What do you need the unsafe here for?

Reply:
That was leftover from the first commit that used a `fixed` buffer, which requires `unsafe`